File: app/src/main/java/tech/thdev/mediaprojectionexample/util/DisplayUtil.java
package tech.thdev.mediaprojectionexample.util;

import android.app.Activity;
import android.util.DisplayMetrics;

/**
 * Created by Tae-hwan on 4/21/16.
 */
public class DisplayUtil {

    public static int getDensityDpi(Activity activity) {
        DisplayMetrics metrics = new DisplayMetrics();
        activity.getWindowManager().getDefaultDisplay().getMetrics(metrics);
        return metrics.densityDpi;
    }
}

File: mediaProjectionLibrary/src/main/java/tech/thdev/media_projection_library/util/DisplayUtil.kt
package tech.thdev.media_projection_library.util

import android.app.Activity
import android.content.Context
import android.util.DisplayMetrics

/**
 * Created by Tae-hwan on 4/21/16.
 * @since 4/12/20 Edit kotlin.
 */
fun Activity.getDensityDpi(): Int {
    return DisplayMetrics().run {
        this@getDensityDpi.windowManager.defaultDisplay.getMetrics(this)
        this.densityDpi
    }
}